airbyte-integrations/connectors/source-intruder/source_intruder/intruder.yaml

```diff
@@ -0,0 +1,70 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["results"]
+  requester:
+    url_base: "https://api.intruder.io/v1"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['access_token'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
```

Comment:
Please implement the pagination https://developers.intruder.io/docs/pagination

Reply:
Done!